.github/workflows/pr.yml

```diff
@@ -1,9 +1,26 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
 name: Run basic tests with Python 3.8
 
-on: [push, pull_request]
+on: [push, pull_request, workflow_dispatch]
```

Comment:
Optional (and could come in a later PR): It might be worth adding a schedule trigger here to run once a day. Since this repo will probably eventually only get occasional contributions that will help us stay on top of any breakages introduced by dependencies.